pages/atom.js

```diff
@@ -0,0 +1,108 @@
+// @flow
+import Layout from "../components/layout/layout";
+import ContentSection from "../components/content-section/content-section";
+import React from "react";
+
+import {
+  PageHeader,
+  PageHeaderLeft,
+  PageHeaderRight
+} from "../components/page-header/page-header";
+
+const InteractiveCoding = () => (
+  <ContentSection>
+    <div className="panes center-vertically">
+      <div className="pane-70 pane">
+        <div className="section-graphic">
+          <video
+            style={{ boxShadow: "0 4px 14px 0 rgba(0,0,0,.1)" }}
+            autoPlay
+            loop
+          >
+            <source src="static/atom/interactive.mp4" type="video/mp4" />
```

Comment:
@lgeiger Using media queries to serve up a mobile version should be fine.

I wonder if there would be a way to adjust which video would be served based off of the screen size similar to how we are serving different download buttons based on which OS you are on. Thoughts @rgbkrk?

Another thing that might be better to do is use [whitewater mobile video](https://github.com/samiare/whitewater-mobile-video) because there are still issues of videos not playing on all mobile devices.